packages/manager/src/features/CloudPulse/Utils/FilterConfig.ts

```diff
@@ -17,13 +32,13 @@ export const LINODE_CONFIG: Readonly<CloudPulseServiceTypeFilterMap> = {
         isMetricsFilter: false,
         name: 'Region',
         neededInServicePage: false,
-        priority: 1,
+        priority: 2,
```

Comment:
no need to change priority for now, since this control have not been established